policy/policy.go

```diff
@@ -257,3 +343,15 @@ type PoliciesList []StagedPolicies
 func (l PoliciesList) IsDefault() bool {
 	return len(l) == 1 && l[0].IsDefault()
 }
+
+// WithDefaultAggregation updates the PoliciesList with default aggregation types
+func (l PoliciesList) WithDefaultAggregation() PoliciesList {
+	for _, sp := range l {
+		for j := range sp.policies {
+			if !sp.policies[j].compressedAggTypes.IsDefault() {
```

Comment:
This updates the list of policies in place to avoid allocating a new slice for it, this function is called before sending the policylist out to the network, so it should be safe to update in place.